src/app/common/stores/applications/SsvWeb/Wallet.store.ts

```diff
@@ -133,7 +133,17 @@ class WalletStore extends BaseStore implements Wallet {
    */
   async connectWalletFromCache() {
     const selectedWallet: any = window.localStorage.getItem('selectedWallet');
-    if (selectedWallet && selectedWallet !== 'undefined') {
+    const walletConnectCondition = selectedWallet === 'WalletConnect' && !!window.localStorage.getItem('walletconnect');
```

Comment:
Move it to WalletStore static const:

```
const PROVIDER_WALLET_CONNECT = 'WalletConnect';
```

If you're using the string to save it in local storage - try if possible also save there this const value.

Reply:
done